hack/operatorhub/templates/csv.tpl

```diff
@@ -312,7 +312,7 @@ spec:
             spec:
               serviceAccountName: elastic-operator
               containers:
-              - image: {{ .OperatorRepo }}:{{ .NewVersion }}
+              - image: {{ .OperatorRepo }}-ubi:{{ .NewVersion }}
```

Comment:
There is another reference to the operator image on line 335. Also I think it would be good if the UBI image was exclusively used for the certified operator. I think for the community operator the scratch based image should be fine.